src/tilde_timing_monitor/README.md

```diff
@@ -64,42 +66,39 @@ See <https://autowarefoundation.github.io/autoware-documentation/main/installati
 
 - prepare path list yaml file (see. config/tilde_path_info.yaml)
 
-```yaml
+````yaml
 /**:
   ros__parameters:
-    required_paths:
-      test:
-        /localization/pose_estimator/for_tilde_interpolator_mtt:
-          {
-            mtype: "tilde_msg/msg/MessageTrackingTag",
-            path_name: "EKF=>NDT",
-            path_i: 0,
-            p_i: 100.0,
-            d_i: 150.0,
-            level: warn,
-          }
-        /localization/pose_estimator/pose_with_covariance:
-          {
-            mtype: "geometry_msgs/msg/PoseWithCovarianceStamped",
-            path_name: "PCL=>NDT",
-            path_i: 1,
-            p_i: 100.0,
-            d_i: 120.0,
-            level: warn,
-          }
-```
-
-| name                | content                             |
-| ------------------- | ----------------------------------- |
-| required_paths      | measurement path information        |
-| mode (ex. test)     | measurement path set id             |
-| topic name          | end point node published topic name |
-| mtype               | message type                        |
-| path_name(any word) | the name given to the path          |
-| path_i              | path number                         |
-| p_i                 | periodic timer value (msec)         |
-| d_i                 | deadline detect timer value (msec)  |
-| level               | diagnostic severity                 |
+    diag_period_sec: 5.0 # frequency of report
+    target_paths:
+      ekf-to-ndt: # path name
+        topic: /localization/pose_estimator/for_tilde_interpolator_mtt # topic name
+        message_type: tilde_msg/msg/MessageTrackingTag # message type
+        severity: warn # severity
+        period: 100.0 # execution frequency of path
+        deadline: 200.0 # deadline of response time
+        violation_count_thresh: 2 # threshold to judge warn or not.
```

Comment:
How aobut "violation_count" or "violation_count_threshold" ?

Reply:
i would like to be  ”violation_count_threshold”.